pkg/dynamicinstrumentation/diconfig/dwarf.go

```diff
@@ -223,6 +223,7 @@ func expandTypeData(offset dwarf.Offset, dwarfData *dwarf.Data, seenTypes map[st
 		if err != nil {
 			return nil, fmt.Errorf("could not find pointer type: %w", err)
 		}
+		typeHeader.ID = randomLabel()
```

Comment:
Should we set these parameters in a separate helper function? That way we don't forget to add the necessary field. It might also be good to add a comment explaining why we're adding a random label here.

Reply:
Added a few comments!